config_defaults_inc.php

```diff
@@ -3618,6 +3618,15 @@
  */
 $g_my_view_boxes_fixed_position = ON;
 
+/**
+ * Toggle whether 'Timeline' boxes show summary of issue after issue ID
+ */
+$g_timeline_show_issue_summary = OFF;
+
+/**
+ * Control the maximum length of each summary shown in timeline when $g_timeline_show_issue_summary is ON.
+ */
+$g_timeline_issue_summary_length = 20;
```

Comment:
Thank you for the guidance. I have added this in 3953b7c. But I do not know how to check whether it works.